infrastructure/prototype-app/template.yaml

```diff
@@ -215,13 +215,39 @@ Resources:
       Name: prototype.solid.integration.account.gov.uk
       ResourceRecords:
         - !GetAtt LoadBalancer.DNSName
-      HostedZoneId: 
+      HostedZoneId:
         Fn::ImportValue:
           Fn::Sub: "${CommonStackName}-SolidRoute53"
       Type: CNAME
       TTL: 3000
 
+  GitHubIdentityProvider:
+    Type: AWS::IAM::OIDCProvider
+    Properties:
+      ClientIdList:
+        - "sts.amazonaws.com"
+      ThumbprintList:
+        - "6938fd4d98bab03faadb97b34396831e3780aea1"
+      Url: "https://token.actions.githubusercontent.com"
+      Tags:
```

Comment:
Nice! Is this a standard set of tags we should be using?

Reply:
I think so.
I wasn't sure what was going on with the ThumbPrint list so I stepped through and verified that was the right hash. Was kinda cool will note below this what I did to prove it.

That URL seems correct.

The rest of the tags are from [here](https://github.com/alphagov/di-devplatform-deploy/blob/main/github-identity/template.yaml). I actually ended up removing two, `Environment` (on the basis we only have one and are not doing build -> integration -> prod for POC), and `System`, which I take to be representing what this is to other parts of DI... again an assumption here that we're less concerned about that 🤔